src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs

```diff
@@ -306,6 +311,18 @@ private IEnumerable<PackageSource> LoadNuGetSources(PackageSourceLocation packag
             PackageSourceProvider packageSourceProvider = new PackageSourceProvider(settings);
             defaultSources = packageSourceProvider.LoadPackageSources().Where(source => source.IsEnabled);
 
+            packageSourceMapping = packageSourceMapping ?? PackageSourceMapping.GetPackageSourceMapping(settings);
+
+            // filter package patterns if enabled            
+            if (_isNuGetTool && packageSourceMapping?.IsEnabled == true)
+            {
+                IReadOnlyList<string> sources = packageSourceMapping.GetConfiguredPackageSources(packageId.ToString());
```

Comment:
So I use we get the configured package sources, but I don't them being used anywhere. 

Am I missing something?

Reply:
Thanks for the comment! I believe the `sources` are referred in line 321 when counting if `sources.Count == 0`